frontend/packages/knative-plugin/src/utils/create-eventsources-utils.ts

```diff
@@ -18,8 +20,38 @@ import * as camelSourceImg from '../imgs/logos/camelsource.svg';
 import * as containerSourceImg from '../imgs/logos/containersource.png';
 import * as cronJobSourceImg from '../imgs/logos/cronjobsource.png';
 import * as kafkaSourceImg from '../imgs/logos/kafkasource.svg';
+import CronJobSection from '../components/add/event-sources/CronJobSection';
+import SinkBindingSection from '../components/add/event-sources/SinkBindingSection';
+import ApiServerSection from '../components/add/event-sources/ApiServerSection';
+import KafkaSourceSection from '../components/add/event-sources/KafkaSourceSection';
+// eslint-disable-next-line import/no-cycle
+import YAMLEditorSection from '../components/add/event-sources/YAMLEditorSection';
```

Comment:
Why have you disabled this eslint rule here? You cannot just disable any eslint rule that you encounter, they're there for a reason. This creates a cyclic dependency. You cannot have the `eventSourceSectionLoader` in the utils. Move it out to a separate component.

Reply:
moved the util out to a separate file